components/static.html

```diff
@@ -48,15 +48,35 @@ <h2 class="md-title">MyUW works best in Firefox and Chrome</h2>
       </div>
     </md-dialog-content>   
     <md-dialog-actions>
-      <md-button ng-click="portal.hideIEAlert = true" class="md-primary md-confirm-button" type="button" ng-click="dialog.hide()" style="text-transform: none; margin: 8px 0 8px 8px; min-height: 36px; min-width: 88px;">Close</md-sbutton>
+      <md-button ng-click="portal.hideIEAlert = true" class="md-primary md-confirm-button" type="button" onclick="createCookie('ieAlertCookie', 'true', 20)" style="text-transform: none; margin: 8px 0 8px 8px; min-height: 36px; min-width: 88px;">Close</md-sbutton>
     </md-dialog-actions>     
   </md-dialog>
 </div>
 <script>
-  // If it's IE, show the alert
+  // If it's IE and the ieAlertCookie is null, show the alert
   if(window.navigator.userAgent.match(/(MSIE|Trident)/)) {
-    document.getElementById('ieAlert').style.display = 'flex';
+    if (readCookie('ieAlertCookie') === null) {
```

Comment:
How about `if (document.cookie.indexOf('ieAlertCookie') === -1) {` as more succinct way to express this? Would that be idiomatic?